content/automate/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/clone_to_vm.yaml

```diff
@@ -10,6 +10,8 @@ object:
   fields:
   - PreProvision:
       value: "/Cloud/VM/Provisioning/StateMachines/Methods/PreProvision_Clone_to_VM#${/#miq_provision.source.vendor}"
+  - CheckProvisioned:
+      on_error: retry_state(status => 'Error Creating VM')
```

Comment:
Same change would be required to the `template.yaml` in the same directory.  One instance is used when you select the template from `Lifecycle -> Provision Instances` and the other when you select the image and use `Lifecycle -> Provision Instances using this image`